projects/ama-sdk/src/lib/api-implementations/acm-api/project-api.ts

```diff
@@ -98,14 +99,20 @@ export class ACMProjectApi implements ProjectApi {
 
     private createProject(backendProject: BackendProject): Project {
         const type = PROJECT,
-            description = '',
-            version = '0.0.1';
+            description = '';
 
         return {
             type,
             ...backendProject,
-            description,
-            version
+            description
         };
     }
+
+    public release(project: Project): Observable<Project> {
+        return this.requestApiHelper
+        .post(`/v1/projects/${project.id}/releases`, {bodyParam: project})
```

Comment:
Why do we need the bodyParam? According to the swagger the only necessary param is the projectId.
If so, we don't need to pass down the whole project, just the projectId which is a string.